tests/plugins/ml/models/upload.yaml

```diff
@@ -0,0 +1,56 @@
+$schema: ../../../../json_schemas/test_story.schema.yaml
+
+description: Test the creation of models using upload API (deprecated).
+distributions:
+  excluded:
+    - amazon-managed
+    - amazon-serverless
+warnings:
+  multiple-paths-detected: false
+epilogues:
+  - path: /_plugins/_ml/models/{model_id}
+    method: DELETE
+    status: [200, 404]
+    parameters:
+      model_id: ${get_completed_task.model_id}
+chapters:
+  - synopsis: Register model.
+    id: upload_model
+    path: /_plugins/_ml/models/_upload
+    method: POST
+    request:
+      payload:
+        name: huggingface/sentence-transformers/msmarco-distilbert-base-tas-b
+        version: 1.0.1
+        model_format: TORCH_SCRIPT
+    response:
+      status: 200
+    output:
+      task_id: payload.task_id
+  - synopsis: Wait to get completed task.
+    id: get_completed_task
+    path: /_plugins/_ml/tasks/{task_id}
+    method: GET
+    parameters:
+      task_id: ${upload_model.task_id}
+    response:
+      status: 200
+      payload:
+        state: COMPLETED
+    output:
+      model_id: payload.model_id
+    retry:
+      count: 5
+      wait: 30000
+  - synopsis: Get model.
```

Comment:
This belongs in `models.yaml`. 

If you need to wait to undeploy the model move it to the epilogue.